client/src/test/java/io/pravega/client/stream/impl/EventStreamWriterTest.java

```diff
@@ -59,11 +60,8 @@ public void testWrite() {
         Mockito.when(controller.getCurrentSegments(scope, streamName)).thenReturn(getSegmentsFuture(segment));
         MockSegmentIoStreams outputStream = new MockSegmentIoStreams(segment);
         Mockito.when(streamFactory.createOutputStreamForSegment(eq(segment), any(), any())).thenReturn(outputStream);
-        EventStreamWriter<String> writer = new EventStreamWriterImpl<>(stream,
-                                                                       controller,
-                                                                       streamFactory,
-                                                                       new JavaSerializer<>(),
-                                                                       config);
+        EventStreamWriter<String> writer = new EventStreamWriterImpl<>(stream, controller, streamFactory,
```

Comment:
You don't have to change these back, but let's try to avoid reformatting changes unless the PR is about reformatting.